iot_verifier/src/poc.rs

```diff
@@ -357,6 +376,24 @@ impl Poc {
         })
     }
 
+    async fn dup_witness_report(
+        &self,
+        witness_report: IotWitnessIngestReport,
+    ) -> Result<IotVerifiedWitnessReport, VerificationError> {
+        Ok(IotVerifiedWitnessReport {
+            received_timestamp: witness_report.received_timestamp,
+            status: VerificationStatus::Invalid,
+            report: witness_report.report,
+            location: None,
+            hex_scale: Decimal::ZERO,
+            // default reward units to zero until we've got the full count of
+            // valid, non-failed witnesses for the final validated poc report
+            reward_unit: Decimal::ZERO,
+            invalid_reason: InvalidReason::InvalidCapability,
```

Comment:
Missing the actual "duplicate" error?

Reply:
yeah proto is not merged, so using this to see if the PR goes green.